i18n.md

```diff
@@ -0,0 +1,54 @@
+# Electron Internationalization
+
+🌎 🌍 🌏
+
+💚 **First off, thanks for taking the time to contribute!** 💚
+
+Anyone is welcome to join in the effort, regardless of technical experience or
+familiarity with the Electron project.
+
+The following is a set of guidelines for contributing to Electron's localization
+effort. These are just guidelines, not rules, so use your best judgment and
+feel free to propose changes to this document in a pull request.
+
+## Code of conduct
+
+This project adheres to the Electron organization's [code of conduct][].
+By participating, you are expected to uphold this code. Please report unacceptable
+behavior to coc@electronjs.org.
+
+## Using Crowdin
+
+Electron's localization effort uses [Crowdin][], an awesome platform for collaborative translation.
+Translations made on Crowdin are periodically pulled into this repository, and changes made to the
+source content is periodically uploaded to Crowdin.
+
+### Translating
+
+Here are some guidelines to keep in mind as a translator:
+
+- Don't open pull requests to translate Electron; instead, do all translations on Crowdin.
+- Do not translate JavaScript keywords like `String`, `Event`, `Array`, `Class`, etc.
+- Do not translate Electron classes, method names, event names, etc.
+- If you find an error in the source English docs, open a pull request on the
+  [electron/electron][] repository, not on the website repository.
```

Comment:
Note for future us, I bet we could make a GitHub actions workflow that automatically closes PRs to `docs/` on this repo with a note to do them upstream.